hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/LogCompactionExecutionHelper.java

```diff
@@ -53,7 +52,7 @@ public class LogCompactionExecutionHelper<T extends HoodieRecordPayload, I, K, O
   @Override
   protected void transitionRequestedToInflight(HoodieTable table, String logCompactionInstantTime) {
     HoodieActiveTimeline timeline = table.getActiveTimeline();
-    HoodieInstant instant = HoodieTimeline.getLogCompactionRequestedInstant(logCompactionInstantTime);
+    HoodieInstant instant = table.getMetaClient().getTimelineLayout().getInstantFactory().getLogCompactionRequestedInstant(logCompactionInstantTime);
```

Comment:
The invocation chain is way to long, maybe just `table.getMetaClient().getInstantFactory`, let's try to limit the instant factory instantatiation APIs to the meta client.

Reply:
+1